test/compression.js

```diff
@@ -661,6 +699,70 @@ function createServer (opts, fn) {
   })
 }
 
+function createHttp2Server (opts, fn) {
+  var _compression = compression(opts)
+  var server = http2.createServer(function (req, res) {
+    req.on('error', function (error) {
+      console.error('An error event occurred on a http2 request.', error)
+    })
+    res.on('error', function (error) {
+      console.error('An error event occurred on a http2 response.', error)
+    })
+    _compression(req, res, function (err) {
+      if (err) {
+        res.statusCode = err.status || 500
+        res.end(err.message)
+        return
+      }
+
+      fn(req, res)
+    })
+  })
+  server.on('error', function (error) {
+    console.error('An error event occurred on the http2 server.', error)
+  })
+  server.on('sessionError', function (error) {
+    console.error('A sessionError event occurred on the http2 server.', error)
+  })
+  server.listen(0, '127.0.0.1')
+  return server
+}
+
+function createHttp2Client (port) {
+  var client = http2.connect('http://127.0.0.1:' + port)
+  client.on('error', function (error) {
+    console.error('An error event occurred in the http2 client stream.', error)
```

Comment:
No one is going to comb through the CI logs after every run or upgrade... do these need to be here vs just letting an error crash the process? If they are preventing an expected error, it shouldn't be logging to the screen, as there should be no extra output when everything was actually OK.